sdk/src/Core/Amazon.Runtime/Internal/Util/StringUtils.cs

```diff
@@ -120,11 +147,29 @@ public static string FromDouble(double value)
             return value.ToString(CultureInfo.InvariantCulture);
         }
 
+        public static string FromDouble(double? value)
```

Comment:
Not super important, but any reason you used `value.HasValue` here but `value != null` below on line 167?

Reply:
I went back through the file and made it consistent to use the `value.HasValue` property.